src/main/java/com/github/fge/jsonpatch/ExtendedJsonPatch.java

```diff
@@ -0,0 +1,66 @@
+package com.github.fge.jsonpatch;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.jsontype.NamedType;
+import com.github.fge.jackson.JacksonUtils;
+import com.github.fge.jsonpatch.operation.*;
+import com.github.fge.msgsimple.bundle.MessageBundle;
+import com.github.fge.msgsimple.load.MessageBundles;
+
+import java.io.IOException;
+
+public class ExtendedJsonPatch
```

Comment:
I think I would have a JsonPatchFactory(mapper), then ExtendedJsonPatchFactory.create() create a JsonPatchFactory with the extended subtypes.  Makes it a bit easier to test and I think it still makes it easy for developers to select "standard" or "extended"